libraries/AP_Common/Bitmask.h

```diff
@@ -20,21 +20,37 @@
 
 class Bitmask {
 public:
-    Bitmask(uint16_t numbits) :
-        numwords((numbits+31)/32) {
+    Bitmask(uint16_t num_bits) :
+        numbits(num_bits),
+        numwords((num_bits+31)/32) {
         bits = new uint32_t[numwords];
+        clearall();
     }
     ~Bitmask(void) {
         delete[] bits;
     }
 
     // set given bitnumber
     void set(uint16_t bit) {
+        if (bit >= numbits) {
```

Comment:
Might be worth a comment stating ignoring setting an invalid bit number.  Ok - so I'm being fussy.